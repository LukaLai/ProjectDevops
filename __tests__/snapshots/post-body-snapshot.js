import { render } from '@testing-library/react'
import { PostBody } from '../../src/app/_components/post-body'

it('renders homepage unchanged', () => {
    const { container } = render(<PostBody content={'test'} />)
    expect(container).toMatchSnapshot()
})