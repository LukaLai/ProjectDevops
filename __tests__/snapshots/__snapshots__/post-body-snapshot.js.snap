// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="max-w-2xl mx-auto"
  >
    <div
      class="markdown"
    >
      test
    </div>
  </div>
</div>
`;
